<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogItemEditDialog</class>
 <widget class="QDialog" name="LogItemEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>299</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#F7F7F7;
opacity:1;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>12</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="minimumSize">
            <size>
             <width>275</width>
             <height>188</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>364</width>
             <height>376</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QTableWidget{
  border: 1px solid #C6C6C6;
  opacity: 1;
  background: #FFFFFF;
  margin: 0;
  gridline-color: #E2E2E2;
}
QHeaderView::section{
    spacing: 10px;
    background-color: #FFFFFF;
    color: #000000;
    border:none;
    border-bottom: 1px solid #E2E2E2;
        border-right: 1px solid #E2E2E2;
    margin:0;
        font: Bold 13px &quot;Segoe UI&quot;;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="autoScrollMargin">
            <number>32</number>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>0</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>120</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>89</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>89</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
             </property>
             <property name="text">
              <string>Add Item</string>
             </property>
             <property name="shortcut">
              <string>Return</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>6</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>26</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>-1</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#FFFFFF;
border:1px solid #ADADAD;
opacity: 1;
padding-left:4px;
font: 13px &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Enter Name...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>4</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>89</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>82</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>6</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnrefreshlist">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>89</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
             </property>
             <property name="text">
              <string>Refresh List</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>12</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>299</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>952</width>
         <height>68</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #EEEEEE;
opacity: 1;
margin: 0;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>12</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btndefaultlist">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
              </property>
              <property name="text">
               <string>System Default Keys</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: BOLD 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>12</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>btnrefreshlist</tabstop>
  <tabstop>btndefaultlist</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
